bootstraptest/test_ractor.rb

```diff
@@ -17,6 +40,31 @@
   end
 }
 
+# Ractor#inspect
+# Return only id and status for main ractor
+assert_equal "#<Ractor:#1 running>", %q{
+  Ractor.current.inspect
+}
+
+# Return id, loc, and status for no-name ractor
+assert_match /^#<Ractor:#([^ ]*?) .+.rb:[0-9]+ blocking>$/, %q{
+  r = Ractor.new { Ractor.recv }
+  r.inspect
+}
+
+assert_match /^#<Ractor:#([^ ]*?) .+:[0-9]+ terminated>$/, %q{
+  r = Ractor.new { '' }
+  r.take
+  sleep 0.1
```

Comment:
we should not use `sleep` here because we run tests on many environments and some machine are slow because of low-spec or many tests run in one machine.